src/libs/ReportScrollManager/index.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import _ from 'underscore';
 
 // This ref is created using React.createRef here because this function is used by a component that doesn't have access
 // to the original ref.
 const flatListRef = React.createRef();
 
+// Whether the user is scrolling the report messages list.
+let hasScrolling = false;
```

Comment:
if this is describing the component _currently_ scrolling, I think `isScrolling` would be a better name

Reply:
But Exlint complaints `Exporting mutable 'let' binding, use 'const' instead.eslint[import/no-mutable-exports](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-mutable-exports.md)`

Reply:
I have done the change. Let me know if it does not look good to you.